server/src/main/java/io/crate/analyze/ExplainAnalyzedStatement.java

```diff
@@ -43,8 +43,10 @@ public class ExplainAnalyzedStatement implements AnalyzedStatement, AnalyzedRela
     private final ProfilingContext context;
     private final List<Symbol> outputs;
     private final RelationName relationName;
+    private final boolean showCosts;
+    private final String columnName = "QUERY PLAN";
```

Comment:
Output columnName in psql is always `QUERY PLAN` on all options. We should follow this.